code/data/fake-events-generator/NCNCA-2016-road-manual.js

```diff
@@ -1,4 +1,28 @@
-[
+const timeData = ["8:00 AM", "8:30 AM", "9:00 AM", "9:20 AM", "10:00 AM", "10:15 AM", "11:40 AM", "12:00 PM", "12:03 PM", "1:00 PM", "1:55 PM", "2:00 PM", "2:18 PM"];
+const promoterData = [
+  "2 Wheel Racing",
+  "Alta Alpina Cycling Club",
+  "Alto Velo",
+  "AntiGravity Cycling",
+  "Back of Nowhere Enterprises",
+  "Berkeley Bicycle Club",
+  "Bicycle Planet",
+  "Bicycles Plus",
+  "Bike Monkey",
+  "Bike Station Aptos",
+  "Brian Joder",
+  "Bubba Melcher",
+  "Cal Aggie Cycling",
+  "Cal Cycling",
+  "California Giant Cycling Team",
+  "CCCX (Central Coast Cyclocross)",
+  "Central Sierra Cyclists",
+  "Chico Corsa Cycling Club",
+  "China Peak Mountain Resort"
+];
```

Comment:
I wonder if this is the list of all promoters that exist on NCNCA for 2016? @Me4ta 
Pretty interesting to know if so. Can you move them to Wiki under Promoters section?